pkg/cmd/cli/cmd/newapp.go

```diff
@@ -609,9 +609,17 @@ func handleRunError(err error, baseName, commandName, commandPath string) error
 	}
 	groups := errorGroups{}
 	for _, err := range errs {
-		transformError(err, baseName, commandName, commandPath, groups)
+		//transformError(err, baseName, commandName, commandPath, groups)
+		f(err, baseName, commandName, commandPath, groups)
 	}
 	buf := &bytes.Buffer{}
+	if len(config.ArgumentClassificationErrors) > 0 {
+		fmt.Fprintf(buf, "Errors occurred while determining argument types:\n")
+		for key, value := range config.ArgumentClassificationErrors {
+			fmt.Fprintf(buf, fmt.Sprintf("\n%s:  %v\n", key, value))
+		}
+		fmt.Fprint(buf, "\nErrors occurred during resource creation:\n")
```

Comment:
Yeah I debated putting the "Errors occurred during resource creation:" in even if there were no ArgumentClassificationErrors, simply to add some context and clarification around the terse kcmdutil.MultipleErrors output.

You've tipped me in that direction.  Part of next push.